alf/examples/rnd_super_mario.gin

```diff
@@ -65,6 +67,7 @@ TrainerConfig.use_tf_functions=True
 TrainerConfig.debug_summaries=True
 TrainerConfig.summarize_grads_and_vars=False
 TrainerConfig.summary_interval=10
+TrainerConfig.update_counter_every_mini_batch=True
```

Comment:
Is this just for debug?

Reply:
For updating the curve more frequently, which is what I always use

Reply:
But this does not have effect for on_policy_trainer.

Reply:
I thought this example uses PPO